tests/test_cli.py

```diff
@@ -0,0 +1,22 @@
+import shlex
+import pyhf_benchmark
+import time
+
+
+def test_version(script_runner):
+    command = "pyhf-benchmark --version"
+    start = time.time()
+    ret = script_runner.run(*shlex.split(command))
+    end = time.time()
+    elapsed = end - start
+    assert ret.success
+    assert pyhf_benchmark.__version__ in ret.stdout
+    assert ret.stderr == ""
+    # make sure it took less than a second
+    assert elapsed < 1.0
```

Comment:
At the moment this test is failing by quite a lot as too many things are being imported all at once. This can be avoided by restructuring the way that the CLI works. @kratsg probably had tips for you here, but I'd also just recommend looking at how things are done in the `pyhf` source.

Reply:
@matthewfeickert I cannot find the solution to this problem in `pyhf` because I fail to trace back the code in `pyhf`. I also search the reason why python imports so long but it seems that the searching result is not what we want.

Reply:
>  I cannot find the solution to this problem in `pyhf` because I fail to trace back the code in `pyhf`.

Yeah, it isn't because of `pyhf` as our CLI is structured to be fast. So I'd recommend that you check out how it is setup there.

> I also search the reason why python imports so long but it seems that the searching result is not what we want.

What did you find so far?

Reply:
@matthewfeickert  [Speeding up the python “import” loader](https://stackoverflow.com/questions/2010255/speeding-up-the-python-import-loader) and [Python — Speed Up Imports?
](https://stackoverflow.com/questions/6025635/python-speed-up-imports) I think the answers are not related to CLI modification. `pyhf` should have a solution to this problem because the source code of test version command is the same as we used in `pyhf-benchmark`

Reply:
> `pyhf` should have a solution to this problem because the source code of test version command is the same as we used in `pyhf-benchmark`

No, I mean look at how the CLI source code for `pyhf` is structured: https://github.com/scikit-hep/pyhf/tree/master/src/pyhf/cli

Reply:
Not sure why it's slow. It looks pretty much the same as we do for pyhf.

Reply:
I noticed in the logs that this was happening

```
E       AssertionError: assert 'Matplotlib i...e a moment.\n' == ''
E         + Matplotlib is building the font cache; this may take a moment.
```

as `matplotlib` was being used for the first time in an install then running once before avoids it.